docs/reports/requirements/1_proj-desc.md

```diff
@@ -62,4 +62,38 @@ Note: the User section will be color-coded to show what's already in place and w
 
 ## PDF Report
 
-I will write my PDF report section here to later be incorporated into the full document.
+### High-level Description (about one paragraph)
+
+The objective for the Speaking Portal Project is to design, develop, and deploy a lip-sync animation (LSA) API for the
+Kukarella Table Reads text-to-speech (TTS) web application. This API will serve as an animation generating add-on for
+this system so that the user can both listen to and watch their avatar speak the user-provided text.
+
+### User groups
+
+In this project, there are two main user groups who will be using the software: general users and developers. The
+general users are the general public who use the Kukarella web application; this could be anyone who would like to
+listen to rather than read some text in any language with a customized avatar. The developers are the Kukarella
+employees who will be responsible for incorporating the API into their current system and adding more customizable
```

Comment:
This is not necessary at all just a lil simpler put:
```suggestion
listen to rather than read some text in any language with a customized avatar. The developers are the Kukarella
employees responsible system maintenance and the addition of more customizable
```